src/resources/styles.ts

```diff
@@ -110,6 +110,11 @@ export const haStyle = css`
     color: var(--app-header-text-color, white);
   }
 
+  app-toolbar,
```

Comment:
This is ok for now. One day we should rip all these styles out of the "generic styles" page.